packages/dev/core/src/Materials/shadowDepthWrapper.ts

```diff
@@ -306,6 +315,7 @@ export class ShadowDepthWrapper {
                 samplers: origEffect.getSamplers(),
                 defines: join + "\n" + origEffect.defines.replace("#define SHADOWS", "").replace(/#define SHADOW\d/g, ""),
                 indexParameters: origEffect.getIndexParameters(),
+                shaderLanguage: (this._baseMaterial as ShaderMaterial).options?.shaderLanguage ?? ShaderLanguage.GLSL,

```

Comment:
`shaderLanguage: origEffect.shaderLanguage` will do it, after the getter has been added to `Effect`.

Reply:
Thanks @Popov72  for review. I didn't like specific ShaderMaterial import in ShadowDepthWrapper, but I didn't know shader language  option is available in Effect class. I have added getter as you suggested.